CHANGES.rst

```diff
@@ -9,7 +9,11 @@ Changes
 `Unreleased <https://github.com/Ouranosinc/Magpie/tree/master>`_ (latest)
 ------------------------------------------------------------------------------------
 
-* Nothing new for the moment.
+* Do not allow users to have the same username that only differs in terms of upper/lowercase
+  Ziggurat foundations assumes that users will not have usernames that differ in terms of case.
+  This means that if the database contains a user with the username "Test" and another with the username "test". The
+  login procedure will not know to differentiate the two.
+  (fixes `#595 <https://github.com/Ouranosinc/Magpie/issues/595>`_).
```

Comment:
Use format ``` `User` ``` and ``` ``user_name`` ``` as employed for the rest of the docs.

Missing dot at the end of the first line.
Move the `(fixes ...)` definition after the first line (after upper/lowercase), no dot in between. 

Move under section 
```rst
Bug Fixes
~~~~~~~~~~~~~~~~~~~~~
```